server/authorization.go

```diff
@@ -354,24 +353,57 @@ func addRequestHeaderTransforms(req *authv3.CheckRequest, envRequest *config.Env
 				}
 			}
 
-			// strip proxy base path from request path
-			addRequestHeader(okResponse, envoyPathHeader, envRequest.GetOperationPath(), false)
-
-			// header transforms from env config
 			transforms := envRequest.GetHTTPRequestTransformations()
-			for _, rhpat := range transforms.RemoveHeaders {
-				rhpat = strings.ToLower(rhpat)
+
+			// http path transformation
+			pathTransform := transforms.PathTransform
+			var targetPath = envRequest.GetOperationPath()
+			if pathTransform != "" {
+				targetPath = envRequest.Reify(pathTransform)
+				targetPath = path.Clean(targetPath)
+			}
+
+			queryMap := envRequest.GetQueryParams()
+			for _, t := range transforms.QueryTransforms.Remove {
+				t = strings.ToLower(t) // TODO: move
```

Comment:
What does this TODO mean?

Reply:
Good question. I should've removed the TODO, but my thought was to move the ToLower() to parsing, but that would add more complexity without much value.